docs/ref/models/indexes.txt

```diff
@@ -32,6 +32,19 @@ references of :class:`Index` which includes the `index options`_.
 
 A list of the name of the fields on which the index is desired.
 
+By default the indexes are created with an ascending order for all the columns.
+To define an index with a descending order for a particular column, a hyphen
+(-) can be added before the field's name while defining the index.
+
+For example::
+
+    Index(fields=['headline', '-pub_date'], name='headline_pub_date_reverse_idx')
+
+The above index would be issue an SQL with ``(headline, pub_date DESC)``. This
+is supported only for PostgreSQL, Oracle and some versions of SQLite. If your
+database doesn't support defining ordering on index columns it will simply be
```

Comment:
I have [modified](https://github.com/django/django/pull/6982/commits/606dda9d9e0908942bbb4cf0b7072374964bbba4) the current test to execute the index creation query irrespective of the database (and test the behaviour only for the backends that support it). So this execution will break if the database doesn't respect nor ignores the `ASC/DESC` passed in the query.
Do you think that would be good enough?